activestate.yaml

```diff
@@ -51,6 +51,15 @@ constants:
   - name: SCRIPT_EXT
     if: ne .OS.Name "Windows"
     value: .sh
+  - name: UPX_TARGET
+    if: eq .OS.Name "Windows"
+    value: ./.github/deps/Windows/bin/upx.exe
+  - name: UPX_TARGET
+    if: eq .OS.Name "MacOS"
+    value: ./.github/deps/macOS/bin/upx
+  - name: UPX_TARGET
+    if: eq .OS.Name "Linux"
+    value: ./.github/deps/Linux/bin/upx
```

Comment:
This should not be necessary due to http://github.com/ActiveState/cli/blob/8de7f9a720b1db6ded36740da29e39e8ccaa4380/.github/workflows/build.yml#L83-L83